company-kata-solutions/src/test/java/org/eclipse/collections/companykata/Exercise6Test.java

```diff
@@ -126,6 +146,11 @@ public void supplierNamesAsTildeDelimitedString()
     @Test
     public void deliverOrdersToLondon()
     {
+        this.company.getCustomers()
+                .asLazy()
+                .selectWith(Customer::livesIn,"London")
+                .flatCollect(Customer::getOrders)
+                .each(Procedures.cast(Order::deliver));
```

Comment:
The `Procedures.cast` should be unnecessary here. Please try and remove. Thanks!

Reply:
Hi Don. Just removed Procedures.cast now. I see it is not required here. Thanks for the tip!